Datasnap/Classes/Event Types/BeaconEvent.m

```diff
@@ -10,9 +10,9 @@
 @implementation BeaconEvent
 @synthesize place;
 @synthesize beacon;
-- (BOOL)validate
+- (BOOL)isValid
```

Comment:
@almcintyre, @DevinJohnson : should this method be part of a protocol that is implemented by the various subclasses of 'EventProperty'?